src/script/lua_api/l_object.cpp

```diff
@@ -353,7 +353,7 @@ int ObjectRef::l_set_wielded_item(lua_State *L)
 	ItemStack item = read_item(L, 2, getServer(L)->idef());
 	bool success = co->setWieldedItem(item);
 	if (success && co->getType() == ACTIVEOBJECT_TYPE_PLAYER) {
-		getServer(L)->SendInventory(((PlayerSAO*)co));
+		getServer(L)->SendInventory((PlayerSAO*)co, true);
```

Comment:
```suggestion
		getServer(L)->SendInventory((PlayerSAO *)co, true);
```

Reply:
if you're changing this anyway it should be a `dynamic_cast<>`
(do we use those anywhere else?)